components/brave_wallet/browser/zcash/zcash_transaction_utils_unittest.cc

```diff
@@ -88,4 +75,67 @@ TEST(ZCashTransactionUtilsUnitTest, PickZCashTransparentInputs) {
   }
 }
 
+#if BUILDFLAG(ENABLE_ORCHARD)
```

Comment:
Is the reason we've got tests behind this flag because it affects code points in a way where tests would fail if this flag isn't on?